src/modules/navigation/partials/NavigationTableItem.vue

```diff
@@ -235,6 +244,18 @@ export default {
 		createView() {
 			emit('tables:view:create', { tableId: this.table.id })
 		},
+
+		exportFile() {
+			axios.get(generateOcsUrl(`/apps/tables/api/2/tables/scheme/${this.table.id}`)).then(res => {
+				const blob = new Blob([JSON.stringify(res.data.ocs.data)], { type: 'application/json' })
```

Comment:
Maybe we can have a non-ocs controller that does the same as the API for exporting and use a Content-Disposition header for forcing to download. Putting this in a fake link seems risky with larger data sets

Reply:
I've created the non OCS endpoint and used the content-disposition header. Sadly I've not been able to somehow trigger the download without a fake link element. I've also not been able to find any solution on how to automatically download files via the browser without such a workaround. Also there's this slightly angry blog about how it's impossible to trigger downloads via the header using axios: https://medium.com/@drevets/you-cant-prompt-a-file-download-with-the-content-disposition-header-using-axios-xhr-sorry-56577aa706d6

Reply:
I've also tried doing the download via a fake form so that the browser handles the request. But when that's done the redirect just returns `csrf failed`. Idk how to send our request tokens etc. via form actions.

Reply:
In that case we can just add `?requesttoken=` with https://github.com/nextcloud-libraries/nextcloud-auth/blob/main/lib/requesttoken.ts#L15 to the URL.